modules/cfem_diffusion/cfem_diffusion_solver.cc

```diff
@@ -336,6 +336,7 @@ Solver::Execute()
                                   TextName(),
                                   KSPCG,
                                   PCGAMG,
+                                  0.,
```

Comment:
Is there a preference in the codebase for `0.` vs `0.0` in our standards? I believe the codebase primarily uses the trailing zero (which is also my preference). Not requesting a change here, but `0.0` would be welcomed where `0.` was introduced in various spots this PR.

Reply:
> Is there a preference in the codebase for 0. vs 0.0 in our standards?

I do not think there is anything in the code standards regarding this.

Reply:
There is a small amount of `0.` or `<number>.` in the code. I will update this PR and send another one to unify this. Plus a change to code standards.